src/20_Components/amp-date-picker.html

```diff
@@ -226,5 +242,131 @@
+    <div class="flex">
+      <div class="ampstart-input inline-block mt1">
+        <input class="border-none p0" id="lb-start" placeholder="Start date" on="tap:lb.open" />
+      </div>
+      <div class="ampstart-input inline-block mt1">
+        <input class="border-none p0" id="lb-end" placeholder="End date" on="tap:lb.open" />
+      </div>
+      <button class="ampstart-btn m1 caps" on="tap:lb-picker.clear">Clear</button>
+    </div>
+    <amp-lightbox id="lb" layout="nodisplay">
+      <button class="ampstart-btn absolute m1 caps" on="tap:lb.close" tabindex="0">Close</button>
+      <div class="align-content-center">
+        <amp-date-picker
+          id="lb-picker"
+          type="range"
+          mode="static"
+          layout="fixed-height"
+          height="360"
+          format="MM/DD/YYYY"
+          on="activate: lb.open;
+              deactivate: lb.close;"
+          start-input-selector="#lb-start"
+          end-input-selector="#lb-end"
+        ></amp-date-picker>
+      </div>
+    </amp-lightbox>
+  </div>
+
+  <!-- ## Fullscreen lightbox date picker -->
+  <!--
+    `amp-date-picker` can also display as a fullscreen view inside a lightbox.
+    The `fullscreen` attribute tells the date picker to take up the space in
+    its container and allow its content to scroll vertically.
+  -->
+  <div>
+      <p>Choose your travel dates</p>
+      <div class="flex">
+        <div class="ampstart-input inline-block mt1">
+          <input class="border-none p0" id="lb-fullscreen-start" placeholder="Start date" on="tap:lb-fullscreen.open" />
+        </div>
+        <div class="ampstart-input inline-block mt1">
+          <input class="border-none p0" id="lb-fullscreen-end" placeholder="End date" on="tap:lb-fullscreen.open" />
+        </div>
+        <button class="ampstart-btn m1 caps" on="tap:lb-fullscreen-picker.clear">Clear</button>
+      </div>
+      <amp-lightbox id="lb-fullscreen" layout="nodisplay">
+        <button class="ampstart-btn m1 absolute caps z3" on="tap:lb-fullscreen.close" tabindex="0">Close</button>
+        <amp-date-picker
+          id="lb-fullscreen-picker"
+          fullscreen
+          layout="fill"
+          mode="static"
+          type="range"
+          number-of-months="12"
+          format="MM/DD/YYYY"
+          on="activate: lb-fullscreen.open;
+              deactivate: lb-fullscreen.close;"
+          start-input-selector="#lb-fullscreen-start"
+          end-input-selector="#lb-fullscreen-end"
+        ></amp-date-picker>
+      </amp-lightbox>
+    </div>
+
+  <!-- ## Date picker in a form-->
+  <!--
+    `amp-date-picker` integrates with AMP forms.
+  -->
+  <form method="post"
+    class="p2"
+    action-xhr="/components/amp-form/submit-form-xhr"
+    target="_top">
+    <amp-date-picker
+      id="static-picker"
+      type="single"
+      mode="static"
+      layout="fixed-height"
+      height="360"
+      format="YYYY-MM-DD"
+      input-selector="#static-picker-input">
```

Comment:
Does the input need to be inside this form?

Reply:
Ah that would make sense. Another copy-paste error.